.github/workflows/node-flow-deploy-release-artifact.yaml

```diff
@@ -173,9 +173,67 @@ jobs:
           repo: hashgraph/hedera-services # ensure we are executing in the hashgraph org
           ref: develop # ensure we are always using the workflow definition from the develop branch
           token: ${{ secrets.GH_ACCESS_TOKEN }}
-          inputs: '{
-            "ref": "${{ inputs.ref }}",
-            "author": "${{ inputs.author }}",
-            "msg": "${{ inputs.msg }}",
-            "sha": "${{ inputs.sha }}"
-          }'
+          inputs: |
+            {
+              "ref": "${{ inputs.ref }}",
+              "author": "${{ inputs.author }}",
+              "msg": "${{ inputs.msg }}",
+              "sha": "${{ inputs.sha }}"
+            }
```

Comment:
Why did this need to be updated?

Reply:
It was changed due to formatting errors. It turned out that only a single tab was needed.